packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -213,11 +214,40 @@ function isInstalled(module) {
     path.join(getPaths().base, 'package.json')
   )
 
-  return Object.hasOwn(
-    {
-      ...dependencies,
-      ...devDependencies,
-    },
-    module
-  )
+  const deps = {
+    ...dependencies,
+    ...devDependencies,
+  }
+
+  if (!deps[module]) {
+    // Check node_modules to see if it exists there (this is mainly to handle
+    // testing setup packages with rwfw project:copy)
+    const nodeModulesPackageJsonPath = path.resolve(
+      __dirname,
+      // Back up to the project root dir node_modules
+      '../../../../../..',
+      module,
+      'package.json'
+    )
```

Comment:
Escape backslash hell:

```suggestion
    const nodeModulesPackageJsonPath = require.resolve(`${module}/package.json`)
```

See the example from yarn's docs here: https://yarnpkg.com/advanced/rulebook/#modules-shouldnt-hardcode-node_modules-paths-to-access-other-modules.

Reply:
Thanks a lot! Made the code much more readable. Love how knowledgable you are about all things yarn 🙂